src/com/links/ressys/core/Reservation.java

```diff
@@ -1,27 +1,26 @@
 package com.links.ressys.core;
 
-import java.time.LocalDateTime;
-import java.util.Date;
+import java.time.LocalDate;
 
 public abstract class Reservation{
 	
-	public abstract CustomerConcrete getCustomer();
+	public abstract Customer getCustomer();
 
-	public abstract void setCustomer(CustomerConcrete customer);
+	public abstract void setCustomer(Customer customer);
 
-	public abstract RoomConcrete[] getRooms();
+	public abstract Room[] getRooms();
 
-	public abstract void setRooms(RoomConcrete[] rooms);
+	public abstract void setRooms(Room[] rooms);
 
 	public abstract int getReservationId();
 
 	public abstract void setReservationId(int reservationId);
 
-	public abstract LocalDateTime getStartDate();
+	public abstract LocalDate getStartDate();

```

Comment:
Ok... Avevamo pensato cose diverse allora. Io aggiungerei anche l'ora, ma come campi separati dalla data. In questo modo, c'è modo effettivamente di sapere quando hanno veramente fatto check-in o check-out...